### ISSUE TITLE
Error when creating Spark Session

### ISSUE THREAD

**Issue by RaSi96** [almond-sh/almond#929]
Greetings everyone,

After successfully integrating almond into my Jupyter environment (thanks to the help of kind souls here), when trying to create a spark session using:
```
val spark = {
  NotebookSparkSession.builder()
    .master("local[*]")
    .getOrCreate()
}
```
I have this error returned: (screenshot attached.)
`java.lang.NoClassDefFoundError: org/apache/xbean/asm7/ClassVisitor`

![Screenshot_20220507_001652](https://user-images.githubusercontent.com/102865950/167199490-c04600b6-6668-44e6-941d-96dc3c45dc55.png)

I searched a little online but a lot of responses say to simply download the respective .jar file and add it to the spark project, however the first time I ran the command it automatically downloaded a few .jar files (that I cannot recall the names of) and I'm unsure how to equate a spark project to a Jupyter Lab environment.

Pardon my repeated questioning but I'm quite new to almond, Spark, Scala, etc., so any help would be greatly appreciated!

**System specs:**
Arch Linux 5.17.5-arch1-1
Hadoop 3.3.2
Spark 3.2.1
Scala 2.12.15
Almond 0.10.0

**What I've done**
1. Opened a Jupyter Notebook with the Scala kernel
2. `import `$ivy.'org.apache.spark::spark-sql:3.2.1'`
3. `import org.apache.log4j.{Level, Logger}`
   `Logger.getLogger("org").setLevel(Level.OFF)`
4. `import org.apache.spark.sql._`

Every step was successful until I tried creating a spark session.

**EDIT:** Before running anything in Jupyter I ensured that the hdfs and yarn were running in the background using `stop-all.sh` and then `start-all.sh`.

**Comment by kiendang** [comment]
It might be because spark 3.x is not supported. almond-spark was built to support spark 2.x and has not been updated.

**Comment by RaSi96** [comment]
Well that's actually kind of hilarious! I assume there really isn't a workaround for this, so might I ask how can I uninstall all of almond? Will `jupyter kernelspec remove scala` remove all of the downloaded dependencies too?

**Comment by paul-brenner** [comment]
I'm confused. The [release notes for Almond v 0.10.1 ](https://github.com/almond-sh/almond/releases/tag/v0.10.1) say that spark 3 support was added in 2020. Am I looking at the wrong thing?

**Comment by alexarchambault** [comment]
I'd advice switching to Almond `0.13.1`, I can't reproduce the issue above with it.

**Comment by RaSi96** [comment]
@paul-brenner, apologies for the very delayed response but I concur, I was a little perplexed at the time as well. Unfortunately I bring no further information to this discussion because I since uninstalled it and switched back to using [spylon-kernel](https://github.com/vericast/spylon-kernel) which still seems to work (at least for the time being).

@alexarchambault, thank you for your recommendation, I'll try it out! I have been tracking [pull #1727](https://github.com/jupyter/docker-stacks/pull/1727) since this issue was mentioned in it because like Bidek56 mentions over there, `spylon-kernel` isn't a viable alternative any more.

I'll be closing this issue now because it seems to have been addressed, thank you!
